### PR TITLE
add uninstall.sh

### PR DESCRIPTION
**Pull request by ywsswy**

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/list.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/put.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/put.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/restore.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/restore.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/rm.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/rm.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/trash.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/trash.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trash_cli-0.17.1.14.egg-info
```
Comment (andreafrancia) [inline uninstall.sh:30]: Thank you for your submission.
These paths are specific for your installation, you can use `pip uninstall trash-cli` to unistall trash-cli

**Comment by andreafrancia** [comment]
Thank you for your patch. Unfortunately it's not needed because you can use `pip uninstall trash-cli` to uninstall.

**Comment by andreafrancia** [comment]
I have updated the README adding the uninstall instructions.

**Comment by ywsswy** [comment]
> Thank you for your patch. Unfortunately it's not needed because you can use `pip uninstall trash-cli` to uninstall.

thank you, but i install from sources `python3 setup.py install --user`.
use `python3 -m pip uninstall trash-cli` can not remove files below:
```
$ ll ~/.local/bin/
total 28
-rwxr-xr-x 1 users 124 Sep  2 15:26 trash
-rwxr-xr-x 1 users 126 Sep  2 15:26 trash-empty
-rwxr-xr-x 1 users 125 Sep  2 15:26 trash-list
-rwxr-xr-x 1 users 124 Sep  2 15:26 trash-put
-rwxr-xr-x 1 users 128 Sep  2 15:26 trash-restore
-rwxr-xr-x 1 users 123 Sep  2 15:26 trash-rm
$
```

**Comment by andreafrancia** [comment]
Run this:
```
rm -fv ~/.local/bin/{trash,trash-empty,trash-list,trash-put,trash-restore,trash-rm}
```

**Pull request closed without merging**